hack/e2e_test.go

```diff
@@ -104,7 +104,7 @@ func TestParse(t *testing.T) {
 		},
 		{
 			[]string{"wut", "--", "-h"},
-			flags{getDefault, oldDefault, []string{"-h"}},
+			flags{getDefault, oldDefault, []string{"--verbose-commands=true", "-h"}},
 			nil,
 		},
```

Comment:
add a testcase to make sure -v=false also works?

Reply:
Done, forgot to comment back here.